src/Transports.AspNetCore/AuthorizationHelper.cs

```diff
@@ -0,0 +1,66 @@
+#nullable enable
+
+using System.Security.Claims;
+using System.Security.Principal;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace GraphQL.Server.Transports.AspNetCore;
+
+/// <summary>
+/// Helper methods for performing connection authorization.
+/// </summary>
+public static class AuthorizationHelper
+{
+    /// <summary>
+    /// Performs connection authorization according to the options set within
+    /// <see cref="AuthorizationParameters{T}"/>.  Returns <see langword="true"/>
```

Comment:
```suggestion
    /// <see cref="AuthorizationParameters{T}"/>. Returns <see langword="true"/>
```

Reply:
All comments properly have two spaces between each sentence.

Reply:
Why do it? Is it a special markup?

Reply:
No, it is simply the correct way to write sentences.  It is unfortunate that HTML does not typically display the second space, but nevertheless, it is correct.

Reply:
>  correct way to write sentences

Really? In english? I hear such for the first time ever. And for all previous years, you have never done that.

Reply:
I've been working with documents (a lot) since all we had was typewriters.  Sentences always had to have two spaces after a period.  It was most certainly wrong if not so, and my work would need to be corrected.  Not everyone does so, and if you search online you may find articles calling this a "dated" style of writing.  Seems ridiculous to me.  It is so much easier to read sentences if they have two spaces between them.  The documents that go out of our office still have two spaces between each sentence.

Btw, Word has an option to check it in any document you write.

![image](https://user-images.githubusercontent.com/6377684/163737393-df73a28d-0d06-45a7-bc13-81b8373f7a20.png)

Interestingly, the articles I read often say the two-space convention came about from the monospaced print by typewriters.  First, I it easier to read proportionally-spaced fonts with two spaces between sentences, and second, our source code is typically read in a monospace font.  So if there ever were a place to use two spaces today, it would be in our xml comments here in source code, where it is seen as a monospaced font, even though the extra space is ignored when displayed by html in a proportionally-spaced font.

Reply:
> And for all previous years, you have never done that.

It would be more accurate to say that I allowed you to change my comments to only have a single space, or I preemptively changed them to a single space in light of the fact that the rest of the repo had single spaces between spaces.  This repo will have 100% new xml comments.  There is no need to stick with a single space between sentences, _especially when it will display identically._

I also do not like the editorconfig and wish to change that.  If you look in my new repository, I use a editorconfig that is different, and personally I think it makes the code much more readable.  Specifically, the opening bracket after an if or try/catch blocks is placed on the same line, like this:

```cs
if (x == true) {
    // stuff
} else {
    // stuff
}

try {
    // stuff
} catch (OperationCancelledException) {
    // stuff
}
```

This makes it so that you can see a lot more code on one page while still being indented properly.  Now, I know that almost nobody does that with C# code.  (The format is more like BASIC and VB code, which I grew up with.)  So, I will grudgingly use the C# convention and as I copy my code over here I reformat it according to the current editorconfig.  Perhaps I will post a PR with the code reformatted at some point, so you can see, but I understand if we want to the typical C# convention for formatting.

Reply:
> Specifically, the opening bracket after an if or try/catch blocks is placed on the same line

No, no, no Java-style brackets. Dozens (if not a hundreds) repositories with the C# code that I saw use exactly the style that is specified in editorconfig now. This is a de facto standard of writing on C#. Look at .NET/CLR/roslyn teams styles and MS guides.

Regarding typewriters. I suspected something like this. Nevertheless, I would not do that, because it is a deviation from the accepted common style used by many C# developers. I also had the preferences (long ago), which violated these agreements. This is a matter of habit.

Reply:
> This is a matter of habit.

Correct.  I can't type two sentences together without two spaces after the period.

Reply:
> No, no, no Java-style brackets

Yeah, I know, I know.  Well, I can do it how I like in my own repositories 😄 .  We can do the standard C# formatting here.